### ISSUE TITLE
I think that I'm here to give some help.

### ISSUE THREAD

**Issue by boddmg** [Simpleyyt/FuckingSmartConfig4Linux#1]
To finish the smartconfig on Linux yourself, you should fix the main problem that a network interface can only listen to only one channel every moment.

**Comment by Simpleyyt** [comment]
@boddmg : Yes, you are right. But I meet another problem. The libpcap library can't capture the 802.11n (HT40) packet. It means that when the  transmission rates of router is more than 300M, it's hard to capture the packet.
